spec/rails_helper.rb

```diff
@@ -100,7 +85,7 @@
     # :rack_test driver's Rack app under test shares database connection
     # with the specs, so continue to use transaction strategy for speed.
     driver_shares_db_connection_with_specs = Capybara.current_driver == :rack_test
-    DatabaseCleaner.strategy = :truncation
+    DatabaseCleaner.strategy = :deletion
```

Comment:
I wonder if this is why the tests seemed slow on my machine. We'll be able to remove `DatabaseCleaner` after upgrading to Rails 5.1, as there was a change [making database connections shared inside of tests](https://github.com/rails/rails/pull/28083).

Reply:
Yeah, I'm hoping this will speed them up.  I have removing Database Cleaner noted as something to do after upgrading to 5.1.